src/ViewInstance.js

```diff
@@ -523,236 +493,80 @@ class ViewInstance extends React.Component {
-                    <Route
-                      path="/inventory/view/"
-                      render={() => (
-                        <MoveItemsContext moveItems={this.moveItems}>
-                          <HoldingsListContainer
-                            instance={instance}
-                            referenceData={referenceTables}
-                            draggable={this.state.isItemsMovement}
-                            droppable
-                          />
-                        </MoveItemsContext>
-                      )}
-                    />
-                  </Switch>
-                )
-                :
-                null
-            }
-
-            <Row>
-              <Col sm={12}>{newHoldingsRecordButton}</Col>
-            </Row>
-
-            <InstanceAdministrativeView
-              id="acc01"
-              instance={instance}
-              instanceStatuses={referenceTables.instanceStatuses}
-              issuanceModes={referenceTables.modesOfIssuance}
-              statisticalCodes={referenceTables.statisticalCodes}
-              statisticalCodeTypes={referenceTables.statisticalCodeTypes}
-            />
-
-            <InstanceTitleData
-              id="acc02"
-              instance={instance}
-              titleTypes={referenceTables.alternativeTitleTypes}
-            />
-
-            <InstanceIdentifiersView
-              id="acc03"
-              identifiers={identifiers}
-              identifierTypes={referenceTables.identifierTypes}
-            />
-
-            <InstanceContributorsView
-              id="acc04"
-              contributors={contributors}
-              contributorTypes={referenceTables.contributorTypes}
-              contributorNameTypes={referenceTables.contributorNameTypes}
-            />
-
-            <InstanceDescriptiveView
-              id="acc05"
-              instance={instance}
-              resourceTypes={referenceTables.instanceTypes}
-              resourceFormats={referenceTables.instanceFormats}
-              natureOfContentTerms={referenceTables.natureOfContentTerms}
-            />
-
-            <InstanceNotesView
-              id="acc06"
-              instance={instance}
-              noteTypes={referenceTables.instanceNoteTypes}
-            />
-
-            <InstanceElecAccessView
-              id="acc07"
-              electronicAccessLines={electronicAccess}
-              electronicAccessRelationships={referenceTables.electronicAccessRelationships}
-            />
+                      </MoveItemsContext>
+                    )}
+                  />
+                </Switch>
+              )
+              :
+              null
+          }
+
+          <Row>
+            <Col sm={12}>{newHoldingsRecordButton}</Col>
+          </Row>
 
-            <InstanceSubjectView
-              id="acc08"
-              subjects={subjects}
-            />
+          {
+            (holdingsrecordid && !itemid)
+              ? (
```

Comment:
```suggestion
              && (
```

Reply:
will skip for now as these blocks will be moved from `ViewInstance` in next PRs